.github/workflows/scenarios-permian.yml

```diff
@@ -20,6 +20,9 @@ jobs:
     env:
       TEST_JOBS: 16
       GITHUB_TOKEN: /home/github/github-token
+      # Can't figure out how to read the timeout-minutes value defined above
+      # so define it again as environment variable usable in run script
+      TIMEOUT_MINUTES: 480
```

Comment:
Maybe something like `${{ fromJSON(jobs.scenario.timeout-minutes) }}` - but advice like that is cheap, testing is the real expense...

Reply:
Oh yes, I tried quite hard to find a way, but for `jobs.scenario.timeout-minutes` `jobs` is undefined and for `job.scenario.timeout-minutes` the value is empty. Nevermind, it is a separate issue applicable also at our current solution.